pkg/webhook/managedresource/managedresource_validating_webhook_test.go

```diff
@@ -0,0 +1,222 @@
+package managedresource
```

Comment:
Please add the microsoft copyright comment https://github.com/Azure/fleet/blob/main/cmd/crdinstaller/main.go#L1-L4